src/commands.ts

```diff
@@ -428,6 +431,34 @@ function openInExternalEditor(packageNode: PackageNode) {
     }
 }
 
+interface IDawrinCompatibleTarget extends vscode.QuickPickItem {
+    value: DarwinCompatibleTarget;
+    label: string;
+}
+
+/**
+ * Switches the target SDK to the platform selected in a QuickPick UI.
+ */
+async function switchPlatform() {
+    const onSelect = async (picked: IDawrinCompatibleTarget) => {
+        const sdkForTarget = await SwiftToolchain.getSdkForTarget(picked.value);
+        if (sdkForTarget) {
+            configuration.sdk = sdkForTarget;
+        } else {
+            console.warn("unable to obtain new SDK path");
```

Comment:
You should probably show an error dialog here, users will not see the console output
```suggestion
            vscode.window.showErrorMessage("Unable to obtain SDK path")
```

Reply:
That's a better idea, thanks. I forgot how buried away the console is when you aren't actively debugging an extension.